content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -197,24 +197,18 @@ and [`emptydir`](/docs/concepts/storage/volumes/#emptydir).
 
 ## Delegating volume permission and ownership change to CSI driver
 
-{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
 If you deploy a [Container Storage Interface (CSI)](https://github.com/container-storage-interface/spec/blob/master/spec.md)
 driver which supports the `VOLUME_MOUNT_GROUP` `NodeServiceCapability`, the
 process of setting file ownership and permissions based on the
 `fsGroup` specified in the `securityContext` will be performed by the CSI driver
-instead of Kubernetes, provided that the `DelegateFSGroupToCSIDriver` Kubernetes
-feature gate is enabled. In this case, since Kubernetes doesn't perform any
+instead of Kubernetes. In this case, since Kubernetes doesn't perform any
 ownership and permission change, `fsGroupChangePolicy` does not take effect, and
 as specified by CSI, the driver is expected to mount the volume with the
 provided `fsGroup`, resulting in a volume that is readable/writable by the
 `fsGroup`.
 
-Please refer to the [KEP](https://github.com/gnufied/enhancements/blob/master/keps/sig-storage/2317-fsgroup-on-mount/README.md)
-and the description of the `VolumeCapability.MountVolume.volume_mount_group`
-field in the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume)
-for more information.
-
```

Comment:
I believe this paragraph was meant to give users a reference for the implementation details. I'm removing this because it doesn't look very useful here.

Reply:
@bertinatto Can we add a link to the CSI driver doc here: https://kubernetes-csi.github.io/docs/support-fsgroup.html#delegate-fsgroup-to-csi-driver?

Reply:
Hi @xing-yang; I don't mind linking that page here, but if I understand https://github.com/kubernetes/website/pull/37700#pullrequestreview-1168772463 correctly, we should make this page self-sufficient and avoid linking other docs here. Do you think we're missing some info here?

Reply:
It's OK to link to https://kubernetes-csi.github.io/docs/ because https://kubernetes-csi.github.io/docs/ is also documentation (KEPs, specifications etc are relevant as references, but aren't really documentation).